Podfile

```diff
@@ -211,7 +211,7 @@ abstract_target 'Apps' do
 
     pod 'Gridicons', '~> 1.1.0'
 
-    pod 'WordPressAuthenticator', '~> 1.39.0-beta.1'
+    pod 'WordPressAuthenticator', '~> 1.39.0'
```

Comment:
The diff for the code freeze also had [a bump in WordPressKit](https://github.com/wordpress-mobile/WordPress-iOS/pull/16763/files#diff-8f7d6adf31268a2d897ee34bd170592648d6e520aa237104395e4a4438af50cbL50-R50). That value is no longer valid because `develop` moved to the beta of the next version (from `4.36.0` to `4.37.0-beta.1`)

https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/Podfile#L50